src/main/java/com/commercetools/sync/CliRunner.java

```diff
@@ -0,0 +1,151 @@
+package com.commercetools.sync;
+
+import static java.lang.String.format;
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+import java.util.concurrent.CompletionStage;
+import javax.annotation.Nonnull;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CliRunner {
+  static final String SYNC_MODULE_OPTION_SHORT = "s";
+  static final String HELP_OPTION_SHORT = "h";
+  static final String VERSION_OPTION_SHORT = "v";
+
+  static final String SYNC_MODULE_OPTION_LONG = "sync";
+  static final String HELP_OPTION_LONG = "help";
+  static final String VERSION_OPTION_LONG = "version";
+
+  static final String SYNC_MODULE_OPTION_TYPE_SYNC = "types";
+  static final String SYNC_MODULE_OPTION_PRODUCT_TYPE_SYNC = "productTypes";
+  static final String SYNC_MODULE_OPTION_CATEGORY_SYNC = "categories";
+  static final String SYNC_MODULE_OPTION_PRODUCT_SYNC = "products";
+  static final String SYNC_MODULE_OPTION_INVENTORY_ENTRY_SYNC = "inventoryEntries";
+
+  static final String SYNC_MODULE_OPTION_DESCRIPTION =
+      format(
+          "Choose which sync module to run: \"%s\", \"%s\", \"%s\", \"%s\" or \"%s\".",
+          SYNC_MODULE_OPTION_TYPE_SYNC,
+          SYNC_MODULE_OPTION_PRODUCT_TYPE_SYNC,
+          SYNC_MODULE_OPTION_CATEGORY_SYNC,
+          SYNC_MODULE_OPTION_PRODUCT_SYNC,
+          SYNC_MODULE_OPTION_INVENTORY_ENTRY_SYNC);
+  static final String HELP_OPTION_DESCRIPTION = "Print help information to System.out.";
+  static final String VERSION_OPTION_DESCRIPTION = "Print the version of the application.";
+
+  static final String APPLICATION_DEFAULT_NAME = "COMMERCETOOLS-SYNC";
```

Comment:
Why not: commercetools-project-sync?

Reply:
https://github.com/commercetools/commercetools-project-sync/blob/2-Add-syncer-cli/src/main/java/com/commercetools/project/sync/CliRunner.java#L45